### ISSUE TITLE
Shift-deleting a lot of tiles at once in editor crashes game

### ISSUE THREAD

**Issue by kaitlynia** [ddnet/ddnet#5463]
What I did:

1. Selected all game layers
2. Shift-deleted a large region (I selected a very tall region - around 400 tiles - and my cursor position ended left of the grid)

**Comment by kaitlynia** [comment]
Update: I have not been able to reproduce this. Here's the log:
```
-------------------

Error occurred on Monday, June 20, 2022 at 23:32:54. (with offset 0000000000400000)

DDNet.exe caused an Access Violation at location 000000000051576F in module DDNet.exe Reading from location 0000000017E8F000.

AddrPC           Params
000000000051576F 00000000C1B42600 0000000000000000 00000000469F54E0  DDNet.exe!0x11576f
00000000005193DB 000000004C2986F1 000000000000000A 0000000025C55C40  DDNet.exe!0x1193db
000000000050B3B5 00000000000008C0 0000000003110BAA 0000000002000001  DDNet.exe!0x10b3b5
000000000050C2B6 000000000062FF30 00000000004366D8 00000000C00000BB  DDNet.exe!0x10c2b6
000000000050CF89 0000000000000000 0000000000000000 0000000000000000  DDNet.exe!0x10cf89
0000000000426B10 000000000095FBE0 00000000006998D0 00007FF8E5C35940  DDNet.exe!0x26b10
0000000000619EA8 0000000000000001 0000000001138250 000000000074A488  DDNet.exe!0x219ea8
00000000004013B1 0000000000000000 0000000000000000 0000000000000000  DDNet.exe!0x13b1
00000000004014C6 0000000000000000 0000000000000000 0000000000000000  DDNet.exe!0x14c6
00007FF8E5A57C24 0000000000000000 0000000000000000 0000000000000000  KERNEL32.DLL!BaseThreadInitThunk
00007FF8E6CCD4D1 0000000000000000 0000000000000000 0000000000000000  ntdll.dll!RtlUserThreadStart

DDNet.exe
ntdll.dll   	6.2.18362.1171
KERNEL32.DLL	6.2.18362.1110
KERNELBASE.dll	6.2.18362.1237
ADVAPI32.dll	6.2.18362.752
msvcrt.dll  	7.0.18362.1
sechost.dll 	6.2.18362.1237
RPCRT4.dll  	6.2.18362.1237
discord_game_sdk.dll
SHELL32.dll 	6.2.18362.1237
WS2_32.dll  	6.2.18362.387
ucrtbase.dll	6.2.18362.1110
ole32.dll   	6.2.18362.1082
cfgmgr32.dll	6.2.18362.387
combase.dll 	6.2.18362.1237
shcore.dll  	6.2.18362.1237
windows.storage.dll	6.2.18362.1171
bcryptPrimitives.dll	6.2.18362.1237
msvcp_win.dll	6.2.18362.1110
GDI32.dll   	6.2.18362.1256
profapi.dll 	6.2.18362.693
powrprof.dll	6.2.18362.1
UMPDC.dll
shlwapi.dll 	6.2.18362.1
win32u.dll  	6.2.18362.1237
USER32.dll  	6.2.18362.1256
gdi32full.dll	6.2.18362.1256
kernel.appcore.dll	6.2.18362.1171
cryptsp.dll 	6.2.18362.1
avutil-57.dll	57.17.100.0
bcrypt.dll  	6.2.18362.1237
avcodec-59.dll	59.18.100.0
avformat-59.dll	59.16.100.0
libcurl.dll
CRYPT32.dll 	6.2.18362.592
MSASN1.dll  	6.2.18362.1
swresample-4.dll	4.3.100.0
libopusfile.dll
swscale-6.dll	6.4.100.0
sqlite3.dll
OPENGL32.dll	6.2.18362.387
libwinpthread-1.dll	1.0.0.0
ktmw32.dll  	6.2.18362.1
VERSION.dll 	6.2.18362.1
steam_api.dll	5.92.36.75
SDL2.dll    	2.0.16.0
IMM32.dll   	6.2.18362.387
OLEAUT32.dll	6.2.18362.1110
SETUPAPI.dll	6.2.18362.1
libfreetype.dll	2.11.1.0
libogg.dll
GLU32.dll   	6.2.18362.387
WINMM.dll   	6.2.18362.1
CRYPTBASE.DLL	6.2.18362.1
WINMMBASE.dll	6.2.18362.1
dxcore.dll  	6.2.18362.1
zlib1.dll   	1.2.11.0
libopus.dll
vulkan-1.dll	1.3.204.1
dxgi.dll    	6.2.18362.1049
exchndl.dll 	0.9.4.0
PSAPI.DLL   	6.2.18362.1
mgwhelp.dll 	0.9.4.0
dbghelp.dll 	10.0.18362.1
dbgcore.DLL 	10.0.18362.1
rsaenh.dll  	6.2.18362.1
MSWSOCK.dll 	6.2.18362.815
secur32.dll 	6.2.18362.1
SSPICLI.DLL 	6.2.18362.1049
iphlpapi.dll	6.2.18362.1
uxtheme.dll 	6.2.18362.449
nvoglv64.dll	30.0.14.9649
WTSAPI32.dll	6.2.18362.1
cryptnet.dll	6.2.18362.1
drvstore.dll	6.2.18362.1171
devobj.dll  	6.2.18362.387
wldp.dll    	6.2.18362.1237
WINTRUST.dll	6.2.18362.1237
ntmarta.dll 	6.2.18362.1
MSCTF.dll   	6.2.18362.1198
clbcatq.dll 	2001.12.10941.16384
TextInputFramework.dll	6.2.18362.1171
CoreUIComponents.dll	6.2.18362.1110
CoreMessaging.dll	6.2.18362.1049
wintypes.dll	6.2.18362.1237
iertutil.dll	11.0.18362.1198
graphics-hook64.dll	1.7.1.0
ow-graphics-vulkan.dll	2.2.185.2
owclient.dll	2.2.199.2
SteamOverlayVulkanLayer64.dll
nvspcap64.dll	3.24.0.126
dwmapi.dll  	6.2.18362.267
imagehlp.dll	6.2.18362.1
gpapi.dll   	6.2.18362.1139
NvCameraAllowlisting64.dll	7.1.788.794
nvapi64.dll 	30.0.14.9649
WINSTA.dll  	6.2.18362.836
MMDevApi.dll	6.2.18362.387
avrt.dll    	6.2.18362.1
AUDIOSES.DLL	6.2.18362.1256
resourcepolicyclient.dll	6.2.18362.959
DNSAPI.dll  	6.2.18362.1171
NSI.dll     	6.2.18362.1171
rasadhlp.dll	6.2.18362.1
fwpuclnt.dll	6.2.18362.1110
schannel.DLL	6.2.18362.1082
mskeyprotect.dll	6.2.18362.1
ncrypt.dll  	6.2.18362.1
NTASN1.dll  	6.2.18362.1
ncryptsslp.dll	6.2.18362.1
symsrv.dll  	10.0.18362.1
wininet.dll 	11.0.18362.1082

Windows 6.2.9200
DrMingw 0.9.4
```

**Comment by def-** [comment]
What's the exact version you were using?

@Jupeyy Maybe we could print out the exact DDNet version string in drmingw crash dump too? Last line looks like a good fit for that.

**Comment by Jupeyy** [comment]
Would be helpful indeed

**Comment by kaitlynia** [comment]
Yes sorry, 16.1

**Comment by Robyt3** [comment]
```                                                                                  
src/game/editor/layer_tiles.cpp:54                                                                                       
src/game/editor/layer_tiles.cpp:441                                                                               
src/game/editor/editor.cpp:2632
```

on https://github.com/ddnet/ddnet/tree/16.1

**Comment by Robyt3** [comment]
Fixed by #5701. I can reproduce the crash before this PR but not with it.
